docs/setup/install.md

```diff
@@ -17,6 +17,27 @@ VERSION=v0.1.0
 kubectl apply -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
 ```
 
+## Install a custom-configured released version
+
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+First, download the release's `manifests.yaml` file:
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+
+Second, with your favourite editor open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
```

Comment:
Probably worth an example?

Reply:
should we add that in a separate config.md file?

Reply:
similar to https://kubernetes.io/docs/reference/scheduling/config/

Reply:
That would make sense as we add more configuration complexity in the future. But for now, I'm just not sure the way you are referring to the data entry is clear enough. So an example would clarify what users should look for.

Maybe there could be a phrase such as:

```
The contents of the ConfigMap are similar to the following:
```

Reply:
> That would make sense as we add more configuration complexity in the future

My thinking is that we need to emphasize the importance of the `manageJobsWithoutQueueName` flag. May be we should just leave it to the API docs?